### PR TITLE
zwave: allow manual field entry

### PR DESCRIPTION
**Pull request by billyzoellers**
- Updating zwave-js to 9.2.1
- Support devices that have both "choices" and allow values to be filled in

### CODE REVIEW

File: plugins/zwave/src/CommandClasses/SettingsToConfiguration.ts [hunk 1]
```diff
@@ -38,7 +38,10 @@ export class SettingsToConfiguration extends ZwaveDeviceBase implements Settings
         const node = this.instance.getNodeUnsafe();
         const metadata = node.getValueMetadata(valueId);
         if ((metadata as any).states) {
-            value = Object.entries((metadata as any).states).find(([, v]) => v === value)?.[0];
+            const state_value = Object.entries((metadata as any).states).find(([, v]) => v === value)?.[0];
+            if (state_value)
+                Object.entries((metadata as any).states).find(([, v]) => v === value)?.[0];
+            this.console.log(value)
```
Comment (koush) [inline plugins/zwave/src/CommandClasses/SettingsToConfiguration.ts:44]: should remove this logging I think

File: plugins/zwave/src/CommandClasses/SettingsToConfiguration.ts [hunk 2]
```diff
@@ -14,13 +14,13 @@ export class SettingsToConfiguration extends ZwaveDeviceBase implements Settings
...
         for (var valueId of values) {
             const metadata = node.getValueMetadata(valueId);
-
             let setting: Setting = {};
             setting.key = valueId.property.toString();
             setting.title = metadata.label;
             setting.description = metadata.description;
             if ((metadata as any).states)
                 setting.choices = Object.values((metadata as any).states);
+                setting.combobox = true;
```
Comment (koush) [inline plugins/zwave/src/CommandClasses/SettingsToConfiguration.ts:23]: indent? was this meant to be under the if or always applied?

Comment (koush) [reply plugins/zwave/src/CommandClasses/SettingsToConfiguration.ts:24]: why would you want free text entry? I'm not sure that's valid

Comment (billyzoellers) [reply plugins/zwave/src/CommandClasses/SettingsToConfiguration.ts:24]: This was intentional. i.e. "make it a combobox if there are states(choices)"

**Comment by billyzoellers** [comment]
@koush fixed to use the allowManualEntry field

**Pull request merged**